src/validation/proof_of_work.cairo

```diff
@@ -0,0 +1,10 @@
+use cairo_verifier::structs::stark_config::ProofOfWorkConfig;
+use cairo_verifier::common::math::Felt252PartialOrd;
+
+const MIN_PROOF_OF_WORK_BITS: felt252 = 30;
+const MAX_PROOF_OF_WORK_BITS: felt252 = 50;
+
+fn proof_of_work_config_validate(config: ProofOfWorkConfig) {
+    assert(MIN_PROOF_OF_WORK_BITS <= config.n_bits, 'Too few bits for proof of work');
+    assert(config.n_bits < MAX_PROOF_OF_WORK_BITS, 'Too many bits for proof of work');
+}
```

Comment:
Inside modules there is/should be config.cairo file where module structs are defined and their validation methods.
I opt for moving this functionality to contextually relevant module WDYT?

Reply:
This has been solved on #42 I believe?